aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/LambdaDestinationEvent.java

```diff
@@ -36,20 +39,31 @@
 public class LambdaDestinationEvent implements Serializable, Cloneable {
 
     private String version;
-    private DateTime timestamp;
+
+    // looking into removing jsonFormat when this issue will be resolved (https://github.com/FasterXML/jackson-modules-java8/issues/76)
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSX")
+    private ZonedDateTime timestamp;
```

Comment:
`timestamp` parameter requires Moment with time zone (`"2019-11-14T18:16:05.568Z"`), not moment in UTC 

Ref: https://docs.aws.amazon.com/lambda/latest/dg/invocation-async.html"

Reply:
`UTC` = `Z`

Reply:
You're right I'm sorry. What I wanted to say is that right now we have been using Joda `DateTime` for timestamps which is TZ-aware, so `ZonedDateTime` was the correct replacement. 

However, since we're defining timestamps in UTC/GMT, we're fine using `Instant`.